test/headers.js

```diff
@@ -438,6 +438,41 @@ describe('Headers', () => {
             expect(res.headers['x-download-options']).to.equal('noopen');
             expect(res.headers['x-content-type-options']).to.equal('nosniff');
         });
+
+        it('does not return the referrer-policy header whe security.referrer is false', async () => {
```

Comment:
It might be good to also ensure that the default behaves like `false`, since it would not likely be picked-up by another test.

Reply:
https://github.com/hapijs/hapi/pull/3775/files#diff-fecfa4a97c0ff7750b8c1e1549f7f3faR306 like this?

Reply:
@devinivy added another test for the default and fixed that `whe` typo.